### ISSUE TITLE
YokoTextField does not update when text is set programmatically

### ISSUE THREAD

**Issue by mstorch** [raulriera/TextFieldEffects#32]
Hi @raulriera, nice work on these effects, they look and function great!!

Noticed one issue when using YokoTextField.  If you set the text on the field after it is already visible, the field does not animate and still shows as if the text is empty.

To reproduce, simply pre-populate a text field in viewDidAppear(animated) instead of viewWillAppear(animated) and the text field won't show the "populated" appearance until you click on it.  Let me know if you need an example project to show it, happy to help!

Thanks a lot.

**Comment by raulriera** [comment]
Hi,

Yeah several appear to be broken when using viewDidAppear, but not with viewDidLoad.... I will try to see where the problem is.

Thanks

**Comment by raulriera** [comment]
```swift
override public var text: String? {
        didSet {
            if let _ = text {
                animateViewsForTextEntry()
            } else {
                animateViewsForTextDisplay()
            }
        }
    }
```

You for the moment subclass them and use that, I will test this out further and see if there is a better way

**Comment by mstorch** [comment]
I'll try it out. Thanks!

**Comment by raulriera** [comment]
Fixed in release 0.4.4
